docs/source/guide/Issues.lrad

```diff
@@ -1,4 +1,4 @@
-# Issues machine
+# Issues machine tutorial
```

Comment:
Maybe `Issue machine` as it is also singular in `rad issue` and `issue-remote.rad`